helm-charts/codegen/gaudi-values.yaml

```diff
@@ -13,10 +13,10 @@ image:
   # Overrides the image tag whose default is the chart appVersion.
   # tag: "1.0"
 
-port: 6666
+port: 7778
```

Comment:
According to the original change, it defined an environment var for customized port number and set 7778 as default:
MEGA_SERVICE_PORT = os.getenv("MEGA_SERVICE_PORT", 7778)

Although it's less possible user would define his own port value, it's better to support this choice to pass the "MEGA_SERVICE_PORT" to container in this template.
codegen/templates/deployment.yaml

Reply:
since I meet with error when adding this env, I will suggest we temporally comment the environment setting in this PR.

Reply:
Issue has been submitted to GenAIExamples: https://github.com/opea-project/GenAIExamples/issues/261